homeassistant/components/google_maps/device_tracker.py

```diff
@@ -46,26 +44,29 @@ class GoogleMapsScanner:
     def __init__(self, hass, config: ConfigType, see) -> None:
         """Initialize the scanner."""
         from locationsharinglib import Service
-        from locationsharinglib.locationsharinglibexceptions import InvalidUser
+        from locationsharinglib.locationsharinglibexceptions import \
+            InvalidCookies
 
         self.see = see
         self.username = config[CONF_USERNAME]
-        self.password = config[CONF_PASSWORD]
         self.max_gps_accuracy = config[CONF_MAX_GPS_ACCURACY]
+        self.scan_interval = timedelta(seconds=config[CONF_SCAN_INTERVAL])
```

Comment:
Use `dict.get` when looking up the config key and put a default interval as default value.

Reply:
Why do we need to make the update interval configurable? Is this change necessary for the new credentials mechanism?

Reply:
Not a requirement of the updated library, just added to allow user to configure frequency--based on comments in the forum and the custom_component version that was created.  It can be removed if it needs to be.

Reply:
I think it's ok this time. Please take for habit to not mix different features or fixes in the same PR. We like to keep PRs as small as possible.